### ISSUE TITLE
A Box office attendant should be able to issue ticket(s) from the available inventory

### ISSUE THREAD

**Issue by Krakaw** [big-neon/bn-api#182]
Child of big-neon/docs#16
Blocked by #504
They must be able to capture the required fields for that ticket type

**Comment by Timo614** [comment]
@Krakaw noticed this was closed, is it no longer needed? I have a PR for showing box office pricing and allowing box office pricing to be used when adding items to the cart. Should I close that one for now (or leave in a pending state for later?).

**Comment by Krakaw** [comment]
Sorry was worried about that, I thought this had been closed with the latest box office items that had come in
